src/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.md

```diff
@@ -198,7 +198,7 @@ The 'helpLinkUri' value is used to show information when this diagnostic in the
 
 ## RS1016: Code fix providers should provide FixAll support
 
-A CodeFixProvider should provide FixAll support to enable users to fix multiple instances of the underlying diagnostic with a single code fix. See documenation at https://github.com/dotnet/roslyn/blob/master/docs/analyzers/FixAllProvider.md for further details.
+A CodeFixProvider should provide FixAll support to enable users to fix multiple instances of the underlying diagnostic with a single code fix. See [documenation](https://github.com/dotnet/roslyn/blob/master/docs/analyzers/FixAllProvider.md) for further details.
```

Comment:
Giving some thoughts about it and keeping in mind that this file is auto-generated I think that it might be easier to go for your original solution (i.e. `<` + url + `>`).

Reply:
@Evangelink If this is auto-generated, we shouldn't edit it manually. The original code that generates it should be modified.

Reply:
Yes, this is auto-generated. You should instead modify the generator as suggested by @Evangelink  https://github.com/dotnet/roslyn-analyzers/blob/dc649547a3d3a80d4e8caffc7bdd31cf3e70cb92/src/Tools/GenerateDocumentationAndConfigFiles/Program.cs#L271

To re-execute the generator, invoke `msbuild /t:pack` on the entire solution or the selected nuget project within https://github.com/dotnet/roslyn-analyzers/tree/master/nuget

Reply:
I know this is auto-generated. My point was to actually say that as it is auto-generated we could easily do a regex replacement to wrap the url in `<>` but it would be hard to do the change you have done.

Reply:
💭 I think I would expect this form for external links:

```
[Title (site)](link)
```

Where in this case `site` would either say `github.com` or, since it's a different project on the same site, `dotnet/roslyn`.

Reply:
> 
> 
> 💭 I think I would expect this form for external links:
> 
> ```
> [Title (site)](link)
> ```
> 
> Where in this case `site` would either say `github.com` or, since it's a different project on the same site, `dotnet/roslyn`.

This message part is auto generated from resx. It can show outside markdown context. So the message itself can't be updated to use markdown syntax. The code that generates the markdown could possibly find links and convert them, but how it should decide the display text for the link?